device_frame/example/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
Your PR is about the iPhone 12 ratio. Please exclude unrelated changes such as this one.